src/pages/landing/weekend-trading/components/_image-text-switching.tsx

```diff
@@ -127,15 +144,6 @@ const ImageTextSwitching = ({ P2P, reverse, two_title }) => {
                                 <Show.Mobile>
```

Comment:
We can also start refactoring these `Show.Mobile` into `Mobile` from `visibility.tsx` and respectively for the `Show.Desktop` as well 🙏

Reply:
should I do it in this PR?